### PR TITLE
qa/tasks/ceph: wait the osds down before start check the logs

### PR DESCRIPTION
**Pull request by NitzanMordhai**
we are starting to check the logs before all osds are down. that can cause the test to fail, we are missing the excpected failure message.

Fixes: https://tracker.ceph.com/issues/67999

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: qa/suites/upgrade/quincy-x/filestore-remove-check/2 - upgrade.yaml [hunk 1]
```diff
@@ -3,14 +3,13 @@ meta:
...
-      - ceph osd require-osd-release quincy
+    all:
 - print: "**** done install.upgrade of nodes"
+- print: "**** start ceph.restart of all osds"
 - ceph.restart:
-    daemons: [mon.a,mgr.x,osd.0,osd.1,osd.2]
+    daemons: [osd.0,osd.1,osd.2]
     mon-health-to-clog: false
     wait-for-healthy: false
     wait-for-osds-up: false
```
Comment (rzarzynski) [inline qa/suites/upgrade/quincy-x/filestore-remove-check/2 - upgrade.yaml:15]: I don't get how we're waiting _the osds down_. Would you mind to clarify?

Comment (NitzanMordhai) [reply qa/suites/upgrade/quincy-x/filestore-remove-check/2 - upgrade.yaml:15]: @rzarzynski yes, the commit message is not clear enough.. i made the changes.
we didn't wait for the osd to go down before we checked the messages.

Comment (rzarzynski) [reply qa/suites/upgrade/quincy-x/filestore-remove-check/2 - upgrade.yaml:15]: It's still confusing. Let's schedule a quick call to go over this.

**Comment by ljflores** [comment]
jenkins retest this please

**Comment by ljflores** [comment]
Rados approved: https://tracker.ceph.com/projects/rados/wiki/MAIN#httpstrackercephcomissues68534

File: qa/suites/upgrade/quincy-x/filestore-remove-check/0-cluster/start.yaml [hunk 2]
```diff
@@ -30,4 +44,3 @@ roles:
   - mgr.x
   - osd.0
   - osd.1
-  - osd.2
```
Comment (batrick) [inline qa/suites/upgrade/quincy-x/filestore-remove-check/0-cluster/start.yaml:33]: ???

**Pull request merged**